docs/index.md

```diff
@@ -74,7 +74,12 @@ The following items are limitations with the current release of the Red Hat Clou
 
 To use the Red Hat Cloud Services provider inside your Terraform configuration you must meet the following:
 
-* Completed [the ROSA getting started](https://console.redhat.com/openshift/create/rosa/getstarted) requirements
+* You completed [the ROSA getting started](https://console.redhat.com/openshift/create/rosa/getstarted) requirements.
+* You generated an offline [OCM token](https://console.redhat.com/openshift/token/rosa). This token is generated through the Red Hat Hybrid Cloud Console. Do not share this token.
+* You have confirmed that you have [GoLang version 1.20 or newer](https://go.dev/doc/install) installed.
+* You have [Terraform version 1.4.6 or newer](https://developer.hashicorp.com/terraform/downloads) installed.
+* You may chose to use a [configured `*.tfvars` file](docs/terraform-vars.md). This file is a definition sheet for all of your variables.
```

Comment:
This is not prerequisites, its just an additional guide, no?
Lets move this bullet to a more appropriate place

Also the link here is wrong, and it should be `guides/terraform-vars.md`